media/inf/syssetup.inf

```diff
@@ -144,3 +144,21 @@ Control="System\CurrentControlSet\Control"
 
 [Classes]
 HKLM,%CurrentVersion%\Installer,"InstallerLocation",,"%11%"
+
+[DestinationDirs]
+IECopy=16422,Internet Explorer
+
+[Finalize] ; Executed by RunOnce on first normal boot
+AddReg=FinalizeAddReg
+CopyFiles=IECopy
+RegisterDlls=IEReg
+
+[FinalizeAddReg]
+HKLM,SOFTWARE\Microsoft\.NETFramework,InstallRoot,,"%10%\Microsoft.NET\Framework\"
+
+[IECopy]
+iexplore.exe,..\iexplore.exe,,0x00002000
+
+[IEReg]
+16422,Internet Explorer,iexplore.exe
+11,,rundll32.exe,,99,"advpack.dll,DelNodeRunDLL32 ""%10%\iexplore.exe"""
```

Comment:
What does 99 correspond to?

Reply:
Timeout. IIRC the default value is broken so I had to specify one.

Reply:
That's a timeout for the whole command? (I mean, timeout for executing "rundll32.exe advpack.dll, ......" ?)

Reply:
It is WaitForSingleObject(parameter*1000) for the child process yes.

Reply:
> Timeout (in seconds). IIRC the default value is broken (when process parameters are present) so I had to specify one.

For reference, this is this line:
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/setupapi/install.c;hb=d5aca44042bebf2d8d91302bf4cde1c329563a53#l698
so the intent is to make it default to 60 seconds if no value is specified.
(I would suggest you use that one instead of 99 seconds ^^)